src/ui.sortable.multiselection.js

```diff
@@ -178,7 +187,7 @@ angular.module('ui.sortable.multiselection', [])
 
           // Create the helper to act as a bucket for the cloned elements
           var helperTag = item[0].tagName;
-          var helper = angular.element('<' + helperTag + '/>');
+          var helper = angular.element('<' + helperTag + selectionCountAttributes + ' />');
```

Comment:
cool, done!

Thanks for your suggestions.

Umberto

On 28 May 2016 at 09:13, Thodoris Greasidis <notifications@github.com>
wrote:

> var helper = angular.element('<' + helperTag + '/>'); if
> (uiSelectionCount && selectionCount > 1) { helper.addClass('
> ui-selection-count').attr('data-ui-selection-count', selectionCount); }





-- 
Umberto Manganiello